canonical-websites/www.ubuntu.com
# 17.10 takeover build

## Done

- Updated homepage with 17.10 takeover as per [design](https://app.zenhub.com/workspace/o/ubuntudesign/www.ubuntu.com-design/issues/94)
- Brief with content: https://docs.google.com/document/d/1NXayZgtgr7FGARzNymKcoufl72B8jEp_h6zjckprzk4/edit#


## QA

- Check that the homepage (www.ubuntu.com) has a new takeover that matches the brief/design
- Check both large and small screens
- Check that the links connect to the right urls


## Issue / Card

Closes #2280
